docker-compose.yml

```diff
@@ -16,17 +16,16 @@ services:
           - ./example/docker/neo4j/conf:/conf
           - ./example/docker/neo4j/plugins:/plugins
           - ./example/backup:/backup
+          - neo4j_data:/neo4j/data
```

Comment:
we have 3 sets of docker compose file for different purpose. I think let's keep the file name as it is for now and only apply the persistent volume change. WDYT?

Reply:
Totally fine with that! It was more of a convenience thing on my end in the first place 🙂  I'll drop that commit